### ISSUE TITLE
[MAS4.1.2] Name property is not properly defined for 'all card inputs' data.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1345]
**Actual Result:**
When focus lands on the whole data, Narrator is reading irrelevant information.
Name property does not defined properly for “all card inputs” data.

**Expected Result:**
Name property should be properly defined for 'all card inputs' data.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show an Adaptive cards with all types of inputs " button by using Tab key and press Enter on it.
5. New data will upload.
6.  Open Narrator (Ctrl + Win. key + Enter)
7. Navigate to the new uploaded content by using Tab key.
8. Observe that name property does not defined properly for “all card inputs” data and narrator is reading irrelevant information.

**User Impact:**
If Name, State and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[MAS4.1.2_Incorrect name for Whole group.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2579140/MAS4.1.2_Incorrect.name.for.Whole.group.zip)

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
